assets/xml/textures/place_title_cards.xml

```diff
@@ -0,0 +1,287 @@
+<Root>
+    <File Name="g_pn_01">
```

Comment:
Could you add a segment number?
I ask because one of the things that was planned to add to ZAPD's error handler was checking if the segment number was missing in a XML.
I understand that a segment number is not needed here and it probably doesn't even make sense, but it would simplify ZAPD's side to not have special cases like this.